e2e/puppeteer_standalone.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+const puppeteer = require('puppeteer');
+const SERVER_URL = 'http://localhost:3001';
+
+const chromeOptions = {
+  executablePath:process.env.chrome,
+  headless: false
+};
+
+(async function main() {
+  try {
+    const browser = await puppeteer.launch(chromeOptions);
+    const page = await browser.newPage();
+    page.emulate({
+      viewport: {
+        width: 1280,
+        height: 1024
+      },
+      userAgent: ''
+    });
+    await page.goto(SERVER_URL, { waitUntil: 'domcontentloaded' });
+
+    // Run happy path test for submitting a transaction
+    await page.waitForSelector('#test_sender_component');
+    await page.waitForTimeout(1000);
+    await page.click('#test_sender_component').then(() => console.log('-- Open Sender dropdown'));
+    await page.waitForTimeout(1000);
+    const [aliceOption] = await page.$x("//div[contains(., '5sauUX')]");
```

Comment:
I feel this kind of selector is a bit too generic, like it can pretty much match any div, no?

Reply:
This is true. Fortunately it is expected (to what we have - or plan to have afaik) to have only 1 div with the address of Alice visible once the dropdown is clicked. Unfortunately, due to MaterialUI there is not much else that we can do. I have tried adding unique identifiers on the dropdown to identify the specific ones - or we could insert into the "contains" the exact address of Alice to match exact.

In any case this is still an MVP solution for E2E testing and it will and should be further improved.

Reply:
Can we restrict this `div` to some (Grand)parent component? Like `page.$x("//#sender-account div[contains...]")`? At least we will not look on the entire page but within the sender account component.

Reply:
As we discussed - lets just let this for now until we finalize on how we are going to use the puppeteer (CI or... )